api/models/site.js

```diff
@@ -34,6 +44,22 @@ const beforeValidate = (site) => {
   }
 };
 
+
+function isEmptyOrUrl(value, throws = true) {
+  const validUrlOptions = {
+    require_protocol: true,
+    protocols: ['http', 'https'],
```

Comment:
Ah, didn't notice that, cool. I think we may want to open an issue to discuss enforcing an HTTPS requirement, but that's scope creep in the PR?